src/dex-ui-components/base/Labels/MetricLabel.tsx

```diff
@@ -1,21 +1,66 @@
 import { Flex, Text, Skeleton } from "@chakra-ui/react";
+import { Color } from "../../themes";
 
 interface MetricLabelProps {
   label: string;
   value: string;
   isLoading?: boolean;
+  labelLeftIcon?: React.ReactElement;
```

Comment:
We have one challenge in `metric label` the every label in `Manage Voting Power` has different `FontStyles` and `FontColor` so for now I have added optional props for that change. let me know if this works.

Reply:
This is ok for now. I believe the metric label styles should match across the application in the final product but reach out to the design team to confirm.